data/mods/Aftershock/items/comestibles/alienfood.json

```diff
@@ -132,5 +173,406 @@
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien liver",
+    "description": "A porous hard organ. Hazarding a guess from its position in the body, this is the liver.",
+    "price_postapoc": 25,
+    "quench": -4,
+    "calories": 92,
+    "vitamins": [ [ "vitA", 75 ], [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 8 ], [ "vitB", 336 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_liver_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_liver",
+    "looks_like": "meat_cooked",
+    "name": "cooked alien liver",
+    "description": "After some time cooking, the pores of the organ have shrunk, and its much more softer, almost yogurt-like in its consistency.",
+    "parasites": 0,
+    "price_postapoc": 50,
+    "fun": -2,
+    "healthy": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_brain",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": { "str_sp": "raw alien brains" },
+    "description": "Yep, definitely a brain.",
+    "color": "light_gray",
+    "healthy": -15,
+    "price_postapoc": 10,
+    "quench": 10,
+    "fun": -20,
+    "parasites": 15,
+    "calories": 78,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 2 ], [ "calcium", 1 ], [ "iron", 8 ], [ "vitB", 54 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_brain_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_brain",
+    "looks_like": "offal_cooked",
+    "name": { "str_sp": "cooked alien brains" },
+    "description": "Fried and scrambled, this doesn't look the most appetizing, but its edible.",
+    "price_postapoc": 25,
+    "parasites": 0,
+    "healthy": -2,
+    "fun": -8,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien kidney",
+    "description": "You are guessing here, but this is the kidney of the animal. The ammonia-like smell was the biggest hint.",
+    "color": "brown",
+    "healthy": -8,
+    "price_postapoc": 25,
+    "quench": -5,
+    "calories": 54,
+    "vitamins": [ [ "vitA", 6 ], [ "vitC", 2 ], [ "calcium", 0 ], [ "iron", 8 ], [ "vitB", 155 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_kidney_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_kidney",
+    "looks_like": "offal_cooked",
+    "name": "cooked alien kidney",
+    "description": "Yep, still smells lik ammonia.",
+    "healthy": -2,
+    "price_postapoc": 25,
+    "quench": -2,
+    "fun": -3,
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_sweetbread",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien sweetbread",
+    "description": "You got this chunk of meat from the throat of the animal, it looks different.",
```

Comment:
"the animal, " -> "the animal; ". And perhaps "looks different" -> "looks... different"?